Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorWidget.cpp

```diff
@@ -953,6 +956,10 @@ namespace AzToolsFramework
             }
             m_currentAsset = "New Asset";
 
+            // Make sure the saved state key is reset since this could be a file opened with the same property editor isntance
+            m_savedStateKey = AZ::Crc32();
+            m_savedStateKey.Add(&newAssetId, sizeof(AZ::Data::AssetId));
```

Comment:
repeat: A new Crc32 can be calculated and assigned into m_savedStateKey in one statement.
```suggestion
            m_savedStateKey = AZ::Crc32(&newAssetId, sizeof(AZ::Data::AssetId));
```